go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	github.com/mitchellh/mapstructure v1.3.1 // indirect
 	github.com/oklog/run v1.1.0 // indirect
 	github.com/oracle/oci-go-sdk v19.3.0+incompatible // indirect
+	github.com/ory/dockertest v3.3.5+incompatible
```

Comment:
There were not changes to go.sum, meaning we were already indirectly depending on this somewhere anyway.